src/linux/server/Main.cxx

```diff
@@ -89,7 +83,14 @@ main(int argc, char *argv[])
 
     // Create file and console log appenders.
     static plog::ColorConsoleAppender<plog::MessageOnlyFormatter> colorConsoleAppender{};
-    static plog::RollingFileAppender<plog::TxtFormatter> rollingFileAppender(logging::GetLogName("server").c_str());
+    std::string logFilePath = "/usr/local/";

```

Comment:
FYI, no path prefix was previously added such that the current working directory would be used, which when run from systemd, is configurable and can be chosen to whatever we want instead of hard-coding it.